src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,45 +1,107 @@
 import { useState } from 'react';
 import { TextField } from '../TextField';
 
-export const NewMovie = () => {
+interface Movie{
+  title: string;
+  description: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+}
+
+type Add = (value: Movie) => void;
+
+export const NewMovie = ({ onAdd }: { onAdd: Add }) => {
   // Increase the count after successful form submission
   // to reset touched status of all the `Field`s
-  const [count] = useState(0);
+  const [count, setCount] = useState(0);
+  const [titleValue, setTitleValue] = useState('');
+  const [descriptionValue, setDescriptionValue] = useState('');
+  const [imgUrlValue, setImgUrlValue] = useState('');
+  const [imdbUrlValue, setImdbUrlValue] = useState('');
+  const [imdbIdValue, setImdbIdValue] = useState('');
+
+  const movie: Movie = {
+    title: titleValue,
+    description: descriptionValue,
+    imgUrl: imgUrlValue,
+    imdbUrl: imdbUrlValue,
+    imdbId: imdbIdValue,
+  };
+
+  const reset = () => {
+    setCount(count + 1);
+    setTitleValue('');
+    setDescriptionValue('');
+    setImgUrlValue('');
+    setImdbUrlValue('');
+    setImdbIdValue('');
+  };
+
+  const disabledCondition
+  = titleValue === ''
```

Comment:
```suggestion
  = !titleValue
```
can be simplify, check please below also